src/components/SearchResults/UserContact/UserContact.js

```diff
@@ -26,9 +26,9 @@ const UserContact = ({
 
   return (
     <div className={styles.userContact}>
-      {contactData.map((data) => (
+      {contactData.map((data, index) => (
         <a
-          key={data.name}
+          key={index}
```

Comment:
It is preferable to use a unique value that is not an index as the key. You can read in this section of the[ React Docs](https://reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,cause%20issues%20with%20component%20state.) why they advise on not using indexes as keys. The `data.name` you had already worked fine, is there a reason you saw it to be problematic?

Reply:
Yes @gertie-sheshe , `data.name` did become problematic in cases where there is no data available and "Not Available" is returned.. Where there are two unavailable items, the key becomes duplicated, which was what introduced the bug I mentioned.
I know that using index as key is ill-advised but since in this case, the number of items in the array is not changing for every search, I felt it'd be okay.. 
I think an alternative would be to hard code some unique id prop for each of them. what do you think?

Reply:
Ahh I get it now. Valid point. You are right. And yes, I think we meet all the exceptions to the rule in this case so yeah, we can have it as index. Thanks for explaining that @Finniki